src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -88,7 +88,7 @@ public function __invoke(Request $request): JsonResponse
             $targetAdminAccessAction = $formAutocompleteConfig->getAttribute('target_admin_access_action');
         }
 
-        $searchText = $request->get('q');
+        $searchText = $request->get('q','');
```

Comment:
Please, fix CS ([ref](https://flintci.io/repositories/61/analyses/27160).): 
```suggestion
        $searchText = $request->get('q', '');
```